server/views/applications/update_key.njk

```diff
@@ -60,6 +58,7 @@
             </span>
           {% endif %}
         </div>
+        <input type="hidden" value="{{ this_data.keyType }}" name="keyType"/>
```

Comment:
Do you still need this, since you're passing the keyType directly form the request params to the service now?

Reply:
Sorry, I forgot to remove that, will do it now